docs/content/integrations/pandas.mdx

```diff
@@ -3,6 +3,27 @@ title: Dagster with Pandas | Dagster
 description: The dagster-pandas library provides the ability to perform data validation, emit summary statistics, and enable reliable dataframe serialization/deserialization.
 ---
 
+<Note>
```

Comment:
The first thing on every page should be its title - can you move this down to be below the page title?